packages/oruga-next/src/components/autocomplete/Autocomplete.vue

```diff
@@ -797,6 +801,10 @@ function itemOptionClasses(option): ClassBind[] {
 
     return [...itemClasses.value, ...optionClasses.value];
 }
+
+defineExpose({
+    focus,
```

Comment:
Hmmm... Maybe we should consider exposing the focus function for all HTML input based components to make it more consistent? In its own PR ;)

Reply:
This is something that worked prior to 0.8.0, I found the issue because it broke some customer autofocusing logic we have on our end. I think expanding this to cover all focusable elements would be a good idea, custom focus logic is a common use-case.

Reply:
Yes, due to the change in component visibility by switching to the composition API.
Please remove it from this PR and make an issue so I can track it more clearly.

Reply:
Now tracked as https://github.com/oruga-ui/oruga/pull/788